python/graphstorm/config/argument.py

```diff
@@ -1492,6 +1492,18 @@ def eval_metric(self):
 
         return eval_metric
 
+    @property
+    def num_ngnn_layers(self):
```

Comment:
We should think about the naming of ngnn for input layers, ngnn for GNN layers and ngnn for decoders.
How about num_input_ngnn_layers, num_gnn_ngnn_layers and num_decoder_ngnn_layers?

Reply:
What about adding num_encoder_ngnn_layers & num_decoder_ngnn_layers, it can be more clear here with the structure "encoder-gnn-decoder".

Reply:
input and gnn belong to encoder.
You can use num_input_encoder_ngnn_layers and num_gnn_encoder_ngnn_layers.

Reply:
If in that way, that would be too redundant. I prefer still using num_input_ngnn_layers, num_gnn_ngnn_layers and num_decoder_ngnn_layers.

Reply:
> If in that way, that would be too redundant. I prefer still using num_input_ngnn_layers, num_gnn_ngnn_layers and num_decoder_ngnn_layers.

OK